api/api-rule-violations-known.list

```diff
@@ -147,6 +147,7 @@ API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Disks
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,GPUs
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Inputs
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Interfaces
+API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,HyperConvergedConfig,Tolerations
```

Comment:
Are we still missing the right annotation on the map?

Reply:
fixed